src/main/java/Crystal.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Crystal {
+    public static void main(String[] args) {
+        String line;
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+        System.out.println("____________________________________________________________\n"
```

Comment:
Perhaps this string could be turned into a constant to minimise code duplication?